### PR TITLE
Feat/message dont create entities

### PR DESCRIPTION
**Pull request by Kanakala**

### CODE REVIEW

**Review by anttiviljami — APPROVED** [review]
LGTM ✅ nice addition!

File: clients/message-client/src/openapi.json [hunk 1]
```diff
@@ -30,8 +30,17 @@
...
         "summary": "sendMessage",
         "description": "Send an email message",
-        "tags": [
-          "Messages"
+        "tags": null,
+        "parameters": [
+          {
+            "in": "query",
+            "name": "do_not_create_entities",
+            "description": "When true, this flag lets the caller to send only the message and by-pass creating the entities.",
```
Comment (anttiviljami) [inline clients/message-client/src/openapi.json:38]: Why entities (plural) ? What entities normally get created when sending a message. I would've assumed it's just a single message entity.

Comment (Kanakala) [reply clients/message-client/src/openapi.json:38]: It's also the thread entity we are creating while sending a message

Comment (anttiviljami) [reply clients/message-client/src/openapi.json:38]: Got it! Then maybe I'd suggest mentioning these specific entities in the description.

I was immediately wondering if Message API auto-creates contact / file entities too

Comment (Kanakala) [reply clients/message-client/src/openapi.json:38]: sure thing 👍

**Pull request merged**